models/classifiertypeselectors.go

```diff
@@ -1,110 +0,0 @@
-package models
-
-// ClassifierTypeSelectorSummary represents a summary of a classifier type selector.
-type ClassifierTypeSelectorSummary struct {
-	ID   string `json:"id"`
-	Name string `json:"name"`
-}
-
-// ClassifierTypeSelector represents the detail of a classifier type selector.
-type ClassifierTypeSelector struct {
-	ID              string   `json:"id"`
-	Name            string   `json:"name"`
-	ClassifierTypes []string `json:"classifierTypes"`
-}
-
-// AllClassifierTypeSelectors returns all the classifier type selectors for the survey identified by the string surveyID. The classifier type selectors are returned in ascending order.
-func AllClassifierTypeSelectors(surveyID string) ([]*ClassifierTypeSelectorSummary, error) {
-	// We need to run a query first to check if the survey exists so an HTTP 404 can be correctly
-	// returned if it doesn't exist. Without this check an HTTP 204 is incorrectly returned for an
-	// invalid survey ID.
-	err := getSurveyID(surveyID)
-	if err != nil {
-		return nil, err
-	}
-
-	// Now we can get the classifier type selector records.
-	rows, err := db.Query("SELECT classifiertypeselector.id, classifiertypeselector FROM survey.classifiertypeselector INNER JOIN survey.survey ON classifiertypeselector.surveyfk = survey.surveypk WHERE survey.id = $1 ORDER BY classifiertypeselector ASC", surveyID)
-	if err != nil {
-		return nil, err
-	}
-
-	defer rows.Close()
-	classifierTypeSelectorSummaries := make([]*ClassifierTypeSelectorSummary, 0)
-
-	for rows.Next() {
-		classifierTypeSelectorSummary := new(ClassifierTypeSelectorSummary)
-		err := rows.Scan(&classifierTypeSelectorSummary.ID, &classifierTypeSelectorSummary.Name)
-
-		if err != nil {
-			return nil, err
-		}
-
-		classifierTypeSelectorSummaries = append(classifierTypeSelectorSummaries, classifierTypeSelectorSummary)
-	}
-
-	if err = rows.Err(); err != nil {
-		return nil, err
-	}
-
-	return classifierTypeSelectorSummaries, nil
-}
-
-// GetClassifierTypeSelector returns the details of the classifier type selector for the survey identified by the string surveyID and
-// the classifier type selector identified by the string classifierTypeSelectorID.
-func GetClassifierTypeSelector(surveyID string, classifierTypeSelectorID string) (*ClassifierTypeSelector, error) {
-	// We need to run two queries first to check if the survey and classifier type selector both exist
-	// so an HTTP 404 can be correctly returned if the don't exist. Without this check and HTTP 204 is
-	// incorrectly return for an invalid survey ID or classifier type selector ID.
-	err := getSurveyID(surveyID)
-	if err != nil {
-		return nil, err
-	}
-
-	err = getClassifierTypeSelectorID(classifierTypeSelectorID)
-	if err != nil {
```

Comment:
Dunno if it's just github? But there seems to be some odd indentation and formatting - are you running `gofmt`?